### ISSUE TITLE
Add support for additional HTTP headers

### ISSUE THREAD

**Issue by brunogirin** [davidje13/superwstest#10]
Add support for additional HTTP headers in the same way as supertest with a `set` function. This would make it possible to test WS endpoints that require an `Authorization` header for instance. At the moment, I get the following error message when trying to do it the way supertest would:

```
TypeError: request.default(...).ws(...).set is not a function
```

**Comment by brunogirin** [comment]
I now realise that this is related to issue #4 and that the way to do what I need is to pass headers in the options to the `request` function. Take this issue as a query to mimick the `supertest` API in order to increase consistency between the two.

**Comment by davidje13** [comment]
yep, would be good to get API parity here. It's a little complicated to implement (since the header info has to be known before starting the websocket connection, whereas other things in the chain usually happen _after_ it has opened), but should be possible. I'll have a go when I get some time.

**Comment by davidje13** [comment]
Added support for `.set(header, value)`, `.set(headers)` and `.unset(header)` to match the supertest API.

Released as 1.6.0 :)
